client/state/ui/test/reducer.js

```diff
@@ -29,7 +29,8 @@ describe( 'reducer', () => {
 			'olark',
 			'preview',
 			'actionLog',
-			'happychat'
+			'happychat',
+			'mediaModal'
```

Comment:
Perhaps consider opening an eslint PR @dmsnell 😄 I personally dislike it but it does have the downside of making diffs larger.